src/pages/Swap/types.ts

```diff
@@ -0,0 +1,25 @@
+import BigNumber from 'bignumber.js';
+import { Token } from 'types';
+
+interface SwapBase {
+  fromToken: Token;
+  toToken: Token;
+  exchangeRate: BigNumber;
+  direction: 'exactAmountIn' | 'exactAmountOut';
```

Comment:
Since this strings are using in multiple types they could be constants

Reply:
Yeah that's a good point